commands.js

```diff
@@ -366,7 +412,7 @@ var commands = exports.commands = {
 			}
 			if (room.chatRoomData) {
 				room.chatRoomData.modjoin = room.modjoin;
-				Rooms.global.writeChatRoomData();
+				if (!room.isPersonal) Rooms.global.writeChatRoomData();
```

Comment:
As mentioned, this should not be here. If you don't want chat room data to be written, `room.chatRoomData` should not exist in the first place. Just don't create `.chatRoomData` for personal rooms.